docs/_reporting/power-bi/README.md

```diff
@@ -57,67 +57,71 @@ Microsoft offers several ways to analyze and report on your cloud costs. For qui
 
 When you need more advanced reporting or to merge with your own data, we recommend using Microsoft Fabric, Power BI, or a custom or third-party solution. Use the following to determine the best approach for you:
 
-1. For costs under $2-5M in total<sup>1</sup> that don't need savings plan data, you can use the Cost Management connector for Power BI.
```

Comment:
I think we need to keep this. It's still a reality and default expectation for many customers. We need to convey why it's not a good idea.